build.py

```diff
@@ -402,13 +359,16 @@ def RunTests(context,extraArgs):
                     help="Directory where Maya Devkit is installed.")
 
 parser.add_argument("--build-debug", dest="build_debug", action="store_true",
-                    help="Build in Debug mode")
+                    help="Build in Debug mode (default: %(default)s)")
 
 parser.add_argument("--build-release", dest="build_release", action="store_true",
-                    help="Build in Release mode")
+                    help="Build in Release mode (default: %(default)s)")
 
-parser.add_argument("--build-relwithdebug", dest="build_relwithdebug", action="store_true",
-                    help="Build in RelWithDebInfo mode")
+parser.add_argument("--build-relwithdebug", dest="build_relwithdebug", action="store_true", default=True,
+                    help="Build in RelWithDebInfo mode (default: %(default)s)")
```

Comment:
Maybe these should be in a mutually_exclusive_group?

Reply:
Yes that is a good suggestion. I've done. it.